src/conway.js

```diff
@@ -10,7 +21,15 @@ function aliveNeighbors(neighbors) {
  * @return {Boolean}
  */
 function aliveEh(neighbors) {
-
+  if (aliveNeighbors(neighbors) === 0) {
```

Comment:
This should be aliveNeighbors < 2 because with 1 neighbor the cell would still die.